proposals/proposal_selects.py

```diff
@@ -72,15 +70,66 @@ def __init__(self, proposals):
         super().__init__(placeholder="Select a proposal to edit", options=options)
 
     async def callback(self, interaction: discord.Interaction):
-        # Find the selected proposal
         for proposal in self.proposals:
             if proposal["title"] == self.values[0]:
                 selected_proposal = proposal
                 break
         else:
-            await interaction.response.send_message("Proposal not found.", ephemeral=True)
+            await interaction.response.send_message(
+                "Proposal not found.", ephemeral=True
+            )
             return
 
-        # Open the ProposalModal with the selected proposal
         modal = ProposalModal(interaction.channel, selected_proposal)
         await interaction.response.send_modal(modal)
+
+
+class PreviewProposalSelect(discord.ui.Select):
+    def __init__(self, proposals):
+        self.proposals = proposals
+        options = [
+            discord.SelectOption(label=proposal["title"], value=proposal["title"])
+            for proposal in self.proposals
+        ]
+        super().__init__(placeholder="Select a proposal to preview", options=options)
+
+    async def callback(self, interaction: discord.Interaction):
+        for proposal in self.proposals:
+            if proposal["title"] == self.values[0]:
+                selected_proposal = proposal
+                break
+        else:
+            await interaction.response.send_message(
+                "Proposal not found.", ephemeral=True
+            )
+            return
+
+        await interaction.response.send_message(
+            "This is what your proposal will look like upon being published to Discord. The title will be the thread title:",
+            ephemeral=True,
+        )
```

Comment:
line break

Reply:
> line break

Ran black over this file again for fmt, is there something else you meant with your comment?